### ISSUE TITLE
RC Rate

### ISSUE THREAD

**Issue by TechVic-1** [me-no-dev/KissKONG#6]
Can't change rc rate above 1.0, You can decrease it but not increase it above 1.0. I have mine at 1.75.
 don't know if it's the same for rate (expo).

**Comment by me-no-dev** [comment]
Yup... totally my fault.. change this 1000 to 10000: https://github.com/me-no-dev/KissKONG/blob/master/KissKongOSD/KissKongOSD.ino#L879 << 10000 means it will go up to 10.0
